### PR TITLE
Update bug url for failing test

### LINKED ISSUE #92559: Reinstate AbstractXContentFilteringTestCase#testDotsAndDoubleWildcardInExcludedFieldName
`AbstractXContentFilteringTestCase#testDotsAndDoubleWildcardInExcludedFieldName` is marked as `AwaitsFix`, linking #80160, but this issue is closed. We should either remove the AwaitsFix marker, open a new issue to track the bug, or remove the test if it is invalid.

### PR DESCRIPTION
**Pull request by thecoop**
`AbstractXContentFilteringTestCase.testDotsAndDoubleWildcardInExcludedFieldName` is now failing for a different reason, now that jackson is updated.

This closes #92559

### CODE REVIEW

File: libs/x-content/src/test/java/org/elasticsearch/xcontent/support/filtering/AbstractXContentFilteringTestCase.java [hunk 1]
```diff
@@ -355,12 +355,6 @@ private XContentBuilder filter(Builder sample, Set<String> includes, Set<String>
         }
         FilterPath[] excludesFilter = FilterPath.compile(excludes);
         if (excludesFilter != null && Arrays.stream(excludesFilter).anyMatch(FilterPath::hasDoubleWildcard)) {
-            /*
-             * If there are any double wildcard filters the parser based
-             * filtering produced weird invalid json. Just field names
-             * and no objects?! Weird. Anyway, we can't use it.
-             */
-            assertFalse("can't filter on builder with dotted wildcards in exclude", matchFieldNamesWithDots);
```
Comment (DaveCTurner) [inline libs/x-content/src/test/java/org/elasticsearch/xcontent/support/filtering/AbstractXContentFilteringTestCase.java:363]: Did you mean to remove this assertion at the same time?

**Review by DaveCTurner — COMMENTED** [review]
Thanks for picking this up. I left a question.

File: libs/x-content/src/test/java/org/elasticsearch/xcontent/support/filtering/AbstractXContentFilteringTestCase.java (hunk 1 above)
Comment (thecoop) [reply libs/x-content/src/test/java/org/elasticsearch/xcontent/support/filtering/AbstractXContentFilteringTestCase.java:363]: Yeah, the comment no longer applies - it now produces valid JSON, just incorrect.

Comment (DaveCTurner) [reply libs/x-content/src/test/java/org/elasticsearch/xcontent/support/filtering/AbstractXContentFilteringTestCase.java:363]: Ok I see, this is asserting that other tests don't do something known to be bad. Gotcha.

**Review by DaveCTurner — APPROVED** [review]
LGTM

**Pull request merged**